static/templates/admin_tab.handlebars

```diff
@@ -74,6 +74,23 @@
           <button class="button btn-danger" id="do_deactivate_stream_button">{{t "Yes, delete this stream" }}</button>
         </div>
       </div>
+      <div id="realm_aliases_modal" class="modal hide fade" tabindex="-1" role="dialog" aria-labelledby="realm_aliases_modal_label" aria-hidden="true">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
+          <h3 id="realm_aliases_modal_label">{{t "Allowed Domains"}}</h3>
+        </div>
+        <div class="modal-body">
+          <table class="table table-condensed table-stripped" id="alias_table">
+            <th>{{t "Domain"}}</th>
+            <th>{{t "Action"}}</th>
+          </table>
+        </div>
+        <div class="modal-footer centered-footer">
+          <input type="text" id="new_alias"></input>
+          <button type="button" id="add_alias">{{t "Add"}}</button>
```

Comment:
Usually in translation tags we have a space after the closing `"`; can you fix that in this template?